bin/validate_audio.sh

```diff
@@ -1,10 +1,16 @@
 #!/bin/bash
 
-export HTFEED_CONFIG=/l/local/feed/etc/config_audio.yaml
+# Give FEED_HOME env var a default value if unset.
+#
+if [ ! "$FEED_HOME" ] then
```

Comment:
Syntax error -- needs `;` before then. Also prefer `-z` to `!` for absent value checks.